index.js

```diff
@@ -1,24 +1,35 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
+var Funnel = require('broccoli-funnel');
+var MergeTrees = require('broccoli-merge-trees');
+
 module.exports = {
   name: 'ember-sinon',
 
-  options: {
-    nodeAssets: {
-      'sinon': {
-        import: [{ path: 'pkg/sinon.js', type: 'test' }]
-      }
+  included: function(app) {
+    this._super.included.apply(this, arguments);
+
+    while (typeof app.import !== 'function' && app.app) {
+      app = app.app;
```

Comment:
Is this `app` loop still necessary? I thought there was a solution to avoid it now?